lib/modules/datasource/conan/types.ts

```diff
@@ -2,6 +2,14 @@ export interface ConanJSON {
   results?: Record<string, string>;
 }
 
+export interface ConanRevisionJSON {
+  revision?: string;
+}
+
+export interface ConanRevisionsJSON {
```

Comment:
Add tsdoc with link to api docs

Reply:
Unfortunately I did not find any official conan API doc and it does not seem there is any: https://github.com/conan-io/docs/issues/44